guides/web-analytics.md

```diff
@@ -0,0 +1,41 @@
+# Web analytics
+
+We use Google Analytics on most of our sites. It's free ([within limits](https://support.google.com/analytics/answer/1070983?hl=en)), it's easy to implement and there aren't many things it can't track.
+
+The login details for access are kept in 1Password, search for "Google Analytics".
+
+## Implementing
+
+We add Google Analytics via [gtag.js](https://developers.google.com/analytics/devguides/collection/gtagjs/). If you're using [whippet-theme](https://github.com/dxw/whippet-theme), the code is already included and you just have to enter the correct tracking ID.
+
+### Event tracking
+
+If there are specific events you want to track that Google Analytics will not track by default (e.g. clicks on links to external sites, file downloads), you will need to implement a custom event, as per [Google's instructions](https://developers.google.com/analytics/devguides/collection/gtagjs/sending-data).
+
+It's important to note the section on [handling timeouts](https://developers.google.com/analytics/devguides/collection/gtagjs/sending-data#handle_timeouts). If you don't do that, the links you are tracking won't work if GA doesn't load (e.g. if the user has a browser extension that blocks trackers).
```

Comment:
Is this only the case if you follow their guide to the letter and prevent default whenever we want to track an event? What is the benefit of interrupting the normal flow in order to send an event at all? Can we collect more information about the event by having it in a callback? 

As I see it if GA fails to load we won't get notified either way so the following seems like a reliable place to start, but I'm sure there's more capability here I'm missing:

```
  $ ->
    $('.form').click() ->
      gtag('event', 'form_complete', {});
```